website/docs/reference/resource-properties/description.md

```diff
@@ -150,24 +151,81 @@ macros:
 
 <VersionBlock firstVersion="1.9">
 
+You can add a description to a [singular data test](/docs/build/data-tests#singular-data-tests) or a [generic data test](/docs/build/data-tests#generic-data-tests).
+
 <File name='tests/schema.yml'>
 
 ```yml
+# Singular data test example
+
 version: 2
 
 data_tests:
   - name: data_test_name
     description: markdown_string
-
 ```
+</File>
+
+<File name='tests/schema.yml'>
+
+```yml
+# Generic data test example
+
+version: 2
 
+models:
+  - name: model_name
+    columns:
+      - name: column_name
+        tests:
+          - unique
```

Comment:
Oops I forgot a `:` in my example. It should be:
```
models:
  - name: orders
    columns:
      - name: order_id
        tests:
          - unique:
              description: The order_id is unique for every row in the orders model
```

looking at the PR [here](https://github.com/dbt-labs/dbt-core/pull/10850/files)

Reply:
eek great flag, thanks for sharing the code @graciegoheen !